### PR TITLE
Fix breadcrumbs

### PR DESCRIPTION
**Pull request by MarcoPonchia**
#### List of Changes

- align Site Header margin to product header
- fix breadcrumb

#### Motivation and Context

fix breadcrumbs

#### How Has This Been Tested?

User can see breadcrumbs for tutorials and guides pages http://localhost:3000/app-io/guides http://localhost:3000/app-io/tutorials

#### Screenshots (if appropriate):
Screenshot 2023-07-13 at 14 58 40
screenshot-localhost_3000-2023 07 12-10_12_18

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by datalek** [comment]
The screenshot shows that the breadcrumb and page title are not aligned

**Comment by MarcoPonchia** [comment]
> The screenshot shows that the breadcrumb and page title are not aligned

For guides pages retrivered from gitbook i had check with design team this feature and this should be the definitive version mockup will be updated.
For other pages breadcrumbs are now align to container below. Current mismatch of align is due to Abastract  component that is already fixed in other PR.

File: apps/nextjs-website/src/__tests__/fixtures/mockProduct.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Product } from '@/lib/types/product';
+
+export const mockProduct: Product = {
```
Comment (datalek) [inline apps/nextjs-website/src/__tests__/fixtures/mockProduct.ts:3]:
```suggestion
export const aProduct: Product = {
```

File: apps/nextjs-website/src/__tests__/fixtures/mockProduct.ts
Comment (datalek) [inline apps/nextjs-website/src/__tests__/fixtures/mockProduct.ts:1]: Usually we put this kind of information into a `data.ts` file

File: apps/nextjs-website/src/__tests__/helpers/breadcrumbs.helpers.test.ts
Comment (datalek) [inline apps/nextjs-website/src/__tests__/helpers/breadcrumbs.helpers.test.ts:1]: Usually we put tests into the folder of  the file to test. What do you think to move this file under `src/helpers/__tests__/*`?

File: apps/nextjs-website/src/__tests__/helpers/breadcrumbs.helpers.test.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    product,
+    path,
+  });
+  expect(breadcrumbsPaths.length).toEqual(3);
+  expect(breadcrumbsPaths[0].path).toEqual('/');
+  expect(breadcrumbsPaths[0].name).toEqual(homepageTitle);
+  expect(breadcrumbsPaths[1].path).toEqual(product.subpaths.overview.path);
+  expect(breadcrumbsPaths[1].name).toEqual(product.name);
+  expect(breadcrumbsPaths[2].path).toEqual(path);
+  expect(breadcrumbsPaths[2].name).toEqual(product.subpaths.tutorials?.name);
```
Comment (datalek) [inline apps/nextjs-website/src/__tests__/helpers/breadcrumbs.helpers.test.ts:23]:
```suggestion
  const actual = productPageToBreadcrumbs({
    homepageTitle,
    product,
    path,
  });
  const expected = [
    { path: '/', name: homepageTitle },
    { path: product.subpaths.overview.path, name: product.name },
    { path, name: product.subpaths.tutorials?.name },
  ];
  expect(actual).toStrictEqual(expected);
```

File: apps/nextjs-website/src/__tests__/helpers/breadcrumbs.helpers.test.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import {
+  pathToBreadcrumbs,
+  productPageToBreadcrumbs,
+} from '../../helpers/breadcrumbs.helpers';
+import { mockProduct } from '../fixtures/mockProduct';
+import { Path } from '@/lib/types/path';
+
+it('should create breadcrumbs successfully', () => {
```
Comment (datalek) [inline apps/nextjs-website/src/__tests__/helpers/breadcrumbs.helpers.test.ts:8]: Usually we write test in the following form:

```ts
describe('methodName', () => {
  it('should produce a breadcrumb given a product', () => {
      // test here
  });
});
```

File: apps/nextjs-website/src/__tests__/helpers/breadcrumbs.helpers.test.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    path,
+  });
+  expect(breadcrumbsPaths.length).toEqual(3);
+  expect(breadcrumbsPaths[0].path).toEqual('/');
+  expect(breadcrumbsPaths[0].name).toEqual(homepageTitle);
+  expect(breadcrumbsPaths[1].path).toEqual(product.subpaths.overview.path);
+  expect(breadcrumbsPaths[1].name).toEqual(product.name);
+  expect(breadcrumbsPaths[2].path).toEqual(path);
+  expect(breadcrumbsPaths[2].name).toEqual(product.subpaths.tutorials?.name);
+  const paths: readonly Path[] = [
```
Comment (datalek) [inline apps/nextjs-website/src/__tests__/helpers/breadcrumbs.helpers.test.ts:24]: What do you think to move this test on a different `it`?

**Review by datalek — CHANGES_REQUESTED** [review]

**Comment by datalek** [comment]
The activity is tracked by the issue [DEV-859](https://pagopa.atlassian.net/browse/DEV-859) and not yet ready do be developed.

**Pull request closed without merging**